packages/components/molecules/f-user-message/test/specs/accessibility/axe-accessibility.spec.js

```diff
@@ -0,0 +1,15 @@
+import { getAccessibilityTestResults } from '../../../../../../../test/utils/axe-helper';
+const UserMessage = require ('../../../test-utils/component-objects/f-user-message.component');
```

Comment:
Have you run the linter? Not sure it'll like the space after `require`

Reply:
I'll fix this now.